toolchain/sem_ir/formatter.cpp

```diff
@@ -1262,6 +1281,8 @@ class Formatter {
   bool in_terminator_sequence_ = false;
   int indent_ = 0;
   bool after_open_brace_ = false;
+  ConstantId pending_constant_value_ = ConstantId::NotConstant;
```

Comment:
Would it be worth starting to add comments to explain what these members are?

Reply:
Done.